requirements-dev.txt

```diff
@@ -1,6 +1,4 @@
 pytest
 pytest-asyncio
-pytest-flake8
```

Comment:
I'd consider including the base `flake8` package in this list so you can manually run the flake8 test.

Reply:
I wouldn't change it in the action, but it's nice for running on my laptop (I just need to do the latter and can do any testing).